orchagent/pfcactionhandler.cpp

```diff
@@ -521,7 +521,7 @@ PfcWdZeroBufferHandler::~PfcWdZeroBufferHandler(void)
         return;
     }
 
-    sai_object_id_t pg = portInstance.m_priority_group_ids[getQueueId()];
+    sai_object_id_t pg = portInstance.m_priority_group_ids[static_cast <size_t> (getQueueId())];
```

Comment:
>static_cast <size_t> (getQueueId()) [](start = 59, length = 35)

`size_t(getQueueId())` may be more clear.

Reply:
Done